### ISSUE TITLE
Crystal 0.32.0 breaks icr

### ISSUE THREAD

**Issue by lebogan** [crystal-community/icr#114]
Error message when I run make:
```
1 | require "readline"
Error: can't find file 'readline'
```
I had to add crystal-readline as a dependency to get it to work. This is a breaking change for version 0.32.0.

From CHANGELOG:
```
(breaking-change) Remove Readline from std-lib. It's now available as a shard at crystal-lang/crystal-readline (#8364, thanks @ftarulla)
```

Thank you for your attention.

PS - I love this utility!

**Comment by jwoertink** [comment]
Readline was removed, so we will need to include the shard separately.

**Comment by Sija** [comment]
This is fixed by #115.
